service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -131,6 +131,24 @@ void shouldReturnChildrenDetailsWhenPopulatedChildren() {
         assertThat(service.getChildrenDetails(listChildren)).containsOnly(expectedChild);
     }
 
+    @Test
+    void shouldReturnChildrenDetailsWhenPopulatedChildrenDoesNotHaveDob() {
+        List<Element<Child>> listChildren = wrapElements(Child.builder()
+            .party(ChildParty.builder()
+                .firstName("John")
+                .lastName("Smith")
+                .build())
+            .build());
+
+        DocmosisChild expectedChild = DocmosisChild.builder()
+            .name("John Smith")
+            .gender("Male")
+            .dateOfBirth(null)
```

Comment:
Unless you want to be specific, this shouldn't be needed as null is the default value if you don't assign it a value

Reply:
test should not silently assume anything, how do you know if code doesnt set it to "" or "unknown" or "N/A". This test check that it does not

Reply:
Thought so